src/operator/adaptive_avg_pooling.cc

```diff
@@ -91,7 +94,7 @@ static void SpatialAdaptiveAveragePooling_updateGradInput_frame(
           int64_t osizeH,
           int64_t osizeW) {
   int64_t d;
-#pragma omp parallel for private(d)
+#pragma omp parallel for num_threads(engine::OpenMP::Get()->GetRecommendedOMPThreadCount())
```

Comment:
you can still have the private(?) if you need it

Reply:
actually, I think the loop variable is private automatically

Reply:
Thx. I am not familiar with omp, just add the private variable back